### PR TITLE
Db test backfill

### PR DESCRIPTION
**Pull request by jimszalew**
LIST OF CHANGES:
-reopend from previous pull request
-updated several tests to match new db setup
-updated controllers to match new db setup

THINGS I'M UNSURE ON:
-why my tests were green but travis brought up red tests

RSPEC GREEN?
YES
all's well now

*MENTION GROUP MEMBER TO REVIEW PR*
@mimilettd @liambarstad @vaidashi

### CODE REVIEW

**Comment by jimszalew** [comment]
Looks like Travis approves this time! This should take care of testing with the updated database and schema.

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,6 @@
 
 .byebug_history
 /coverage
+
+# Ignore application configuration
+/config/application.yml
```
Comment (mimizotti) [inline .gitignore:23]: Thanks for doing this @jimszalew

**Review by mimizotti — APPROVED** [review]
The back fill tests look good. Thanks for doing this @jimszalew

**Review by vaidashi — APPROVED** [review]
pretty robust seed file, i like it

**Pull request merged**